cirq-google/cirq_google/serialization/circuit_serializer_test.py

```diff
@@ -399,45 +401,34 @@ def test_serialize_deserialize_circuit_with_moments_table():
         circuit=v2.program_pb2.Circuit(
             scheduling_strategy=v2.program_pb2.Circuit.MOMENT_BY_MOMENT,
             moments=[
-                v2.program_pb2.Moment(moment_constant_index=2),
-                v2.program_pb2.Moment(moment_constant_index=3),
+                v2.program_pb2.Moment(moment_constant_index=4),
+                v2.program_pb2.Moment(moment_constant_index=5),
             ],
         ),
         constants=[
             v2.program_pb2.Constant(qubit=v2.program_pb2.Qubit(id='1_1')),
-            v2.program_pb2.Constant(qubit=v2.program_pb2.Qubit(id='1_2')),
             v2.program_pb2.Constant(
-                moment_value=v2.program_pb2.Moment(
-                    operations=[
-                        v2.program_pb2.Operation(
-                            xpowgate=v2.program_pb2.XPowGate(
-                                exponent=v2.program_pb2.FloatArg(float_value=1.0)
-                            ),
-                            qubit_constant_index=[0],
-                        ),
-                        v2.program_pb2.Operation(
-                            xpowgate=v2.program_pb2.XPowGate(
-                                exponent=v2.program_pb2.FloatArg(float_value=1.0)
-                            ),
-                            qubit_constant_index=[1],
-                        ),
-                    ]
+                operation_value=v2.program_pb2.Operation(
+                    xpowgate=v2.program_pb2.XPowGate(
+                        exponent=v2.program_pb2.FloatArg(float_value=1.0)
+                    ),
+                    qubit_constant_index=[0],
                 )
             ),
+            v2.program_pb2.Constant(qubit=v2.program_pb2.Qubit(id='1_2')),
             v2.program_pb2.Constant(
-                moment_value=v2.program_pb2.Moment(
-                    operations=[
-                        v2.program_pb2.Operation(
-                            xpowgate=v2.program_pb2.XPowGate(
-                                exponent=v2.program_pb2.FloatArg(float_value=1.0)
-                            ),
-                            qubit_constant_index=[0],
-                        )
-                    ]
+                operation_value=v2.program_pb2.Operation(
+                    xpowgate=v2.program_pb2.XPowGate(
+                        exponent=v2.program_pb2.FloatArg(float_value=1.0)
+                    ),
+                    qubit_constant_index=[2],
                 )
             ),
+            v2.program_pb2.Constant(moment_value=v2.program_pb2.Moment(operation_indices=[1, 3])),
+            v2.program_pb2.Constant(moment_value=v2.program_pb2.Moment(operation_indices=[1])),
         ],
     )
+    print(serializer.serialize(circuit))
```

Comment:
Did you intend to leave this print statement? I think there's a preference to avoid output noise in the CI due to its monolithic nature. If this is only intended for development, would it work to use a logger with a low severity level (e.g. debug) so that you can enable this only when needed? I'm not sure how logging is set up for cirq/clients.

Reply:
No, this was just for debugging.  Removed.